core/templates/dev/head/editor/ExplorationPreview.js

```diff
@@ -59,5 +61,31 @@ oppia.controller('ExplorationPreview', [
     $scope.$on('playerStateChange', function() {
       $scope.allParams = LearnerParamsService.getAllParams();
     });
+
+    explorationData.getData().then($scope._refreshPreviewTab = function() {
+      var newStateName = explorationInitStateNameService;
+
+      if (angular.equals({}, explorationParamSpecsService.savedMemento)) {
+        newStateName.init(editorContextService.getActiveStateName());
+      }
+      // TODO(oparry): This conditional is a temporary fix meant to prevent
```

Comment:
Generally comments should go immediately above the line which they refer to, not below.